spec/Ingenerator/Mailhook/MailhookSpec.php

```diff
@@ -61,7 +61,9 @@ function its_purge_deletes_dump_file()
 	{
 		$this->given_dump_file('');
 		$this->subject->purge();
-		expect($this->tmp_dir->hasChild(self::DUMP_FILE))->toBe(FALSE);
+		if ($this->tmp_dir->hasChild(self::DUMP_FILE) !== FALSE){
+        	throw new FailureException('Dump file not deleted');
```

Comment:
Just an indentation issue here

Reply:
Yeah, I've tried 3 times to fix it, most of the file is in spaces and phpstorm keeps changing it to tabs. Think the project is a bit of a mix overall

Reply:
Got it sorted